src/Notification/Container.js

```diff
@@ -22,6 +22,13 @@ const NotificationContainer = forwardRef(
       setTimer(new Timer(onDismiss, autoDismissTimeout));
     };
 
+    /**
+     * Extends current timer with new autoDismissTimeout value
+     */
+    const extendTimer = () => {
```

Comment:
I think is not a extend timer but a reset timer with a clear timer then a start timer ?